packages/lib/src/dimension-mapper/utils.ts

```diff
@@ -1,21 +1,32 @@
-import { type Axis, type DimensionMapping } from '@h5web/shared/vis-models';
+import { type DimensionMapping } from '@h5web/shared/vis-models';
 
 export function initDimMapping(
   dims: number[],
   axesCount: number,
+  lockedDimsCount = 0,
```

Comment:
So the dims can only be locked starting from the deepest ones ?

Reply:
Yep, same as the fact that the axes are always mapped to the deepest dimensions. There's an issue open to make that configurable, which will likely require giving `initDimMapping` a more complex signature: #1727. We can deal with locked dimensions as part of this future work.

Note that the RGB visualization currently expects the last dimension to be the channels dimension (otherwise the vis doesn't show up at all), and the new locked dimension feature is used only by the RGB vis for now, so I think the decision to always lock the deepest dimensions is safe and sufficient for a first implementation.

Reply:
Ok, just making sure that I understood well :wink: